samples/xrc/custclas.cpp

```diff
@@ -82,7 +82,7 @@ MyResizableListCtrl::MyResizableListCtrl( wxWindow *parent, wxWindowID id,
     // This listctrl needs to insert its columns in the constructor, since
     // as soon as the listctrl is built, it is resized and grafted onto an
     // "unknown" XRC placeholder. This induces an OnSize() event, calling the
-    // overrriden OnSize function for this class, which needs to have 3
+    // overriden OnSize function for this class, which needs to have 3
```

Comment:
Looks like this typo wasn't quite fixed:
```suggestion
    // overridden OnSize function for this class, which needs to have 3
```